app/models/activity.rb

```diff
@@ -49,6 +49,31 @@ def content_type
 
   attr_accessor :magic_enrolled_reservists
 
+  scope :late_unpayable, lambda {
```

Comment:
Is it possible to write this query as rails functions, or would that get too complex?

Reply:
Get's a bit too complex for the time i have right now